api/src/opentrons/protocol_api/core/engine/deck_conflict.py

```diff
@@ -53,6 +55,10 @@ def __init__(self, message: str) -> None:
 A12_column_front_left_bound = Point(x=-1.8, y=2)
 A12_column_back_right_bound = Point(x=592, y=506.2)
 
+A1_column_front_left_bound = Point(x=-100.8, y=2)
+A1_column_back_right_bound = Point(x=493, y=506.2)
```

Comment:
It sounds like you and Seth have a vision for the proper way to compute these bounds, but until then, would this be more maintainable?

```suggestion
_NOZZLE_PITCH = 9
A1_column_front_left_bound = Point(
    x=A12_column_front_left_bound.x - _NOZZLE_PITCH * 11, y=2
)
A1_column_back_right_bound = Point(
    x=A12_column_front_left_bound.x - _NOZZLE_PITCH * 11, y=506.2
)
```